src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,7 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  margin: 0 auto;
+  width: 1024px;
```

Comment:
CHECKLIST [LAYOUT] - don't hardcode max-width, your catalog container should work fine for any number of cards.
You should  use variables and some calculating for max-width

Reply:
rude mistake fixed and updated! tnx for noticing!